python/smqtk/bin/iqrTrainClassifier.py

```diff
@@ -1,90 +0,0 @@
-"""
```

Comment:
I think we also want to keep this and the `python/smqtk/bin/iqr_app_model_generation.py` script around somewhere. Either in the utils or some `cli` specific sub-module.